src/python/dxpy/scripts/dx.py

```diff
@@ -2989,6 +2989,15 @@ def run_body(args, executable, dest_proj, dest_path, preset_inputs=None, input_n
         "extra_args": args.extra_args
     }
 
+    if any([args.watch or args.ssh or args.allow_ssh]):
+        if run_kwargs["priority"] in ["low", "normal"]:
```

Comment:
The condition should be if `run_kwargs["priority"] != high` (so that includes the case when it is not set at all).

Reply:
If it is unset it would be `None` and set to high in the else below. But it would be easier to read by saying `run_kwargs["priority"] != high`. I will change it in this way and change the check below as well.

Reply:
Also, please check the spec in the JIRA ticket (`(if priority is None, use "default" as the priority value in the warning above)`) and update it accordingly. Thanks!